mobile-widgets/qml/main.qml

```diff
@@ -52,8 +52,8 @@ Kirigami.ApplicationWindow {
 		id: fontMetrics
 		Component.onCompleted: {
 			manager.appendTextToLog("Using the following font: " + fontMetrics.font.family
-				    + " at " + subsurfaceTheme.basePointSize + "pt" +
-				    " with mobile_scale: " + PrefDisplay.mobile_scale)
+					+ " at " + subsurfaceTheme.basePointSize + "pt" +
+					" with mobile_scale: " + PrefDisplay.mobile_scale)
```

Comment:
random whitespace change in the middle of an unrelated commit

Reply:
Corrected, sorry for the problem.